lib/3scale_toolbox/commands/application_command/apply_command.rb

```diff
@@ -0,0 +1,179 @@
+module ThreeScaleToolbox
+  module Commands
+    module ApplicationCommand
+      module Apply
+        class ApplySubcommand < Cri::CommandRunner
+          include ThreeScaleToolbox::Command
+
+          def self.command
+            Cri::Command.define do
+              name        'apply'
+              usage       'apply [opts] <remote> <application>'
+              summary     'update (or create) application'
+              description <<-HEREDOC
+              Update (create if it does not exist) application'
+              \n Application param allows:
+              \n * Application internal id
+              \n * User_key (API key)
+              \n * App_id (from app_id/app_key pair)
+              \n * Client ID (for OAuth and OpenID Connect authentication modes)
+              HEREDOC
+
+              option      nil, 'user-key', 'User Key (API Key) of the application to be created.', argument: :required
+              option      nil, 'application-key', 'App Key(s) or Client Secret (for OAuth and OpenID Connect authentication modes) of the application to be created. Only used when application does not exist.' , argument: :required
```

Comment:
shouldn't this be a symbol as with the other ones?

Reply:
not necessarily